beets/util/__init__.py

```diff
@@ -545,6 +546,16 @@ def hardlink(path, dest, replace=False):
                                   traceback.format_exc())
 
 
+def tmp_path_for(path=''):
```

Comment:
TBH I kind of liked the old name a little more… `tmp_file_for` more clearly suggests that this actually creates a file. Otherwise it might sound more like it computes a path but doesn't touch the filesystem.

Reply:
Yeah that makes sense.  I was going off the idea that `tmp_file_for` implied some kind of file obj would come back, but I get your mental model is too.